worksheet_B/Mandelbrot.cpp

```diff
@@ -15,36 +15,74 @@ int main()
 	const double minX = -2, maxX = 1, minY = -1.5, maxY = 1.5;
 
 	// Generate the image
+	float imageWidth = image.width();
```

Comment:
Why are you mixing `float` and `double`? It's better to stick to one or the other.